internal/clients/client_models.go

```diff
@@ -31,6 +31,9 @@ type InstanceType struct {
 	// The size of the memory, in MiB.
 	MemoryMiB int64 `json:"memory,omitempty"`
 
+	// The total size of the disks, in GB or 0 if EBS only.
+	Storage int64 `json:"storage,omitempty"`
```

Comment:
```suggestion
	StorageGiB `int64 `json:"storage,omitempty"`
```

Reply:
Maybe StorageGB? just to be aligned with AWS?

Reply:
Oh it's storage not memory, so GB it is, sorry my bad: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_InstanceStorageInfo.html